### DISCUSSION TITLE
[BP#7] Respecter le principe de navigation rapide dans l’historique

### DISCUSSION THREAD

**Discussion by tbroyer** [cnumr/best-practices#232] [Bonnes pratiques] [answered]
https://groups.google.com/g/ecoconceptionweb/c/WyyzIK50DlQ/m/MpcOKnhyCQAJ

> le lien avec le "back-forward cache" n'est pas évident pour qui ne connait pas son existence (je soupçonne une large proportion des développeurs web). https://html.spec.whatwg.org/multipage/browsers.html#the-pagetransitionevent-interface & https://developer.mozilla.org/en-US/Firefox/Releases/1.5/Using_Firefox_1.5_caching

Une référence plus à jour serait: https://web.dev/bfcache/

Il faudrait mettre à jour la liste d'exemples pour supprimer les "effets JavaScript" (quasi inexistants en pratique), et peut-être ajouter les conditions qui rendent les pages inéligibles au back-forward cache.

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé la BP pour modification: https://github.com/cnumr/best-practices/issues/266
